corehq/apps/reports/filters/api.py

```diff
@@ -355,7 +357,10 @@ def _create_caseblock(self, case):
             owner_id=self.to_owner,
             case_name=case_name or case.name,
             case_type=case.type,
-            update={**case.case_json, **deid_props},
+            update={
+                self.COMMCARE_CASE_COPY_PROPERTY_NAME: True,
```

Comment:
Does it matter what the value of this property is? If not,  what do you think about having the orignal case ID as the value instead? It could be helpful in scenarios where there are problems with the copied case data. If we have the orignal ID we can then easily track it back to the original case for comparison.

Reply:
Maybe not a bad idea